modules/nw-sctp-enabling.adoc

```diff
@@ -0,0 +1,56 @@
+// Module included in the following assemblies:
+//
+// * networking/using-sctp.adoc
+
+[id="nw-sctp-enabling_{context}"]
+= Enabling Stream Control Transmission Protocol (SCTP)
+
+As a {product-title} cluster administrator, you can load and enable the blacklisted SCTP kernel module on worker nodes in your cluster.
+
+.Prerequisites
+
+* Install the OpenShift Command-line Interface (CLI), commonly known as `oc`.
+* Access to the cluster as a user with the `cluster-admin` role.
+
+.Procedure
+
+. Create a file named `load-sctp-module.yaml` that contains the following YAML definition:
++
+[source,yaml]
+----
+apiVersion: machineconfiguration.openshift.io/v1
+kind: MachineConfig
+metadata:
+  labels:
+    machineconfiguration.openshift.io/role: worker
+  name: load-sctp-module
+spec:
+  config:
+    ignition:
+      version: 2.2.0
+    storage:
+      files:
+        - contents:
+            source: data:,
+            verification: {}
+          filesystem: root
+          mode: 420
+          path: /etc/modprobe.d/sctp-blacklist.conf
+        - contents:
+            source: data:text/plain;charset=utf-8,sctp
+          filesystem: root
+          mode: 420
+          path: /etc/modules-load.d/sctp-load.conf
+----
+
+. To create the MachineConfig object, enter the following command:
++
+----
+$ oc create -f load-sctp-module.yaml
+----
+
+. To wait for the updated configuration to apply, enter the following command:
++
+----
+$ oc wait mcp/worker --for condition=updated
```

Comment:
Not sure what the users should expect from "oc wait mcp/worker --for condition=updated", here is what I got:
[weliang@weliang FILE]$ oc wait mcp/worker --for condition=updated
error: timed out waiting for the condition on machineconfigpools/worker
[weliang@weliang FILE]$ oc wait mcp/worker --for condition=updated
error: timed out waiting for the condition on machineconfigpools/worker
[weliang@weliang FILE]$

Reply:
@weliang1, would watching `oc get clusteroperator/machine-config` for a particular status be a better approach?

Reply:
@jboxman Above cmd is not real useful. "oc get node" can be used to check the node state.
After load load-sctp-module, all the nodes start to reboot. The sctpserver, sctpclient and sctpservice can be created after all the nodes are in Ready state.
[weliang@weliang FILE]$ oc get nodes
NAME                                         STATUS                     ROLES    AGE   VERSION
ip-10-0-137-98.us-east-2.compute.internal    Ready,SchedulingDisabled   worker   79m   v1.16.2
ip-10-0-140-117.us-east-2.compute.internal   Ready                      master   87m   v1.16.2
ip-10-0-155-192.us-east-2.compute.internal   Ready                      worker   79m   v1.16.2
ip-10-0-156-250.us-east-2.compute.internal   Ready                      master   87m   v1.16.2
ip-10-0-161-67.us-east-2.compute.internal    Ready                      master   87m   v1.16.2
ip-10-0-175-145.us-east-2.compute.internal   Ready                      worker   79m   v1.16.2
[weliang@weliang FILE]$ oc get nodes
NAME                                         STATUS   ROLES    AGE   VERSION
ip-10-0-137-98.us-east-2.compute.internal    Ready    worker   82m   v1.16.2
ip-10-0-140-117.us-east-2.compute.internal   Ready    master   90m   v1.16.2
ip-10-0-155-192.us-east-2.compute.internal   Ready    worker   82m   v1.16.2
ip-10-0-156-250.us-east-2.compute.internal   Ready    master   90m   v1.16.2
ip-10-0-161-67.us-east-2.compute.internal    Ready    master   90m   v1.16.2
ip-10-0-175-145.us-east-2.compute.internal   Ready    worker   82m   v1.16.2
[weliang@weliang FILE]$

Reply:
@weliang1, great, thanks!